cilib/run.py

```diff
@@ -41,7 +44,7 @@ def script(script_data, **kwargs):
     os.close(tmp_script[0])
     process = subprocess.Popen(
         ["bash", str(tmp_script_path)],
-        env=env.copy(),
+        env=os.env.copy(),
```

Comment:
Did you mean to use `env` or `env.copy()` or `os.environ.copy()` instead of `os.env.copy()`?

Reply:
Yeah, that should just be `env`.  I don't think the previous version even worked, though.